cmd/spire-server/cli/run/run.go

```diff
@@ -109,6 +110,18 @@ type caSubjectConfig struct {
 	UnusedKeys   []string `hcl:",unusedKeys"`
 }
 
+type federationConfig struct {
+	BundleEndpoint *bundleEndpointConfig         `hcl:"bundle_endpoint"`
+	FederateWith   map[string]federateWithConfig `hcl:"federate_with"`
```

Comment:
We refer in other places (like https://github.com/spiffe/spire/blob/master/proto/spire/common/common.proto#L68 and here https://github.com/spiffe/go-spiffe/blob/master/proto/spiffe/workload/workload.proto#L44) as `federates_with` instead of `federate_with`. I think that we should keep consistency with the `federates_with` name.

Reply:
Sounds good to me.